test/kb_kaiju_server_test.py

```diff
@@ -139,7 +139,7 @@ def prepare_data(cls):
         # Upload Single End Libraries
         #
         cls.SE_reads_refs = []
-        for basename in ['seven_species_nonuniform_10K-PE_reads_', 'seven_species_nonuniform_05K-PE_reads_']:
+        for basename in ['seven_species_nonuniform_10K-PE_reads_', 'seven_species_nonuniform_05K-PE_reads_', 'SE_weird_readIDs_']:
```

Comment:
Awesome

Reply:
So there's already a PE test data piece that matches the read ID pattern in the user ticket?